go.mod

```diff
@@ -1,49 +1,50 @@
 module github.com/tektoncd/triggers
 
-go 1.22
-toolchain go1.22.5
```

Comment:
Can we change toolchain to point to 1.23.8 which is latest insteead of 1.24

Reply:
Updated to go1.23.8